package.json

```diff
@@ -126,7 +126,7 @@
   },
   "homepage": "https://github.com/aragon/ods#readme",
   "engines": {
-    "node": ">=18",
+    "node": ">=18.18.0",
```

Comment:
Hmm, we should check over on the App if everything works as expected with new Node version

Reply:
Yes, we should update the app min node version to 18.18 as well. Given that it is not a major version everything should work fine 👍🏻